### PR TITLE
Updates cypher query in data observation

### PR DESCRIPTION
**Pull request by rewanthtammana**
Signed-off-by: Rewanth Tammana <22347290+rewanthtammana@users.noreply.github.com>

### Description

The cypher query is outdated (possibly due to updated data files). This PR updates the query with a working one!

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ We then run the following commands that finds other containers that share depend
 with the debian image, and counts the number of shared dependencies in descending order.
 
 ```
-MATCH (n:Package{purl:"pkg:oci/debian@sha256:9b0e3056b8cd8630271825665a0613cc27829d6a24906dc0122b3b4834312f7d?repository_url=docker.io/library/debian&tag=latest"}) -[:Contains|DependsOn*1..5]->(d)<-[*1..3]-(o:Package)
+MATCH (n:Package{purl:"pkg:oci/debian:latest?repository_url=docker.io/library",digest:["sha256:9b0e3056b8cd8630271825665a0613cc27829d6a24906dc0122b3b4834312f7d"]}) -[:Contains|DependsOn*1..5]->(d)<-[*1..3]-(o:Package)
```
Comment (pxp928) [inline SETUP.md:172]: Will have to look at this. This should not work anymore.

Comment (rewanthtammana) [reply SETUP.md:172]: It seems to work. I should've attached POC to description.

### Current query - No records found

```
MATCH (n:Package{purl:"pkg:oci/debian@sha256:9b0e3056b8cd8630271825665a0613cc27829d6a24906dc0122b3b4834312f7d?repository_url=docker.io/library/debian&tag=latest"}) -[:Contains|DependsOn*1..5]->(d)<-[*1..3]-(o:Package)
where "CONTAINER" in o.tags
WITH o.purl AS target, collect(d.name) as shared_dep
RETURN target, SIZE(shared_dep) as num_deps, shared_dep
ORDER BY num_deps desc;
```

image

### Updated query - 16 records found

```
MATCH (n:Package{purl:"pkg:oci/debian:latest?repository_url=docker.io/library",digest:["sha256:9b0e3056b8cd8630271825665a0613cc27829d6a24906dc0122b3b4834312f7d"]}) -[:Contains|DependsOn*1..5]->(d)<-[*1..3]-(o:Package)
where "CONTAINER" in o.tags
WITH o.purl AS target, collect(d.name) as shared_dep
RETURN target, SIZE(shared_dep) as num_deps, shared_dep
ORDER BY num_deps desc;
```

image

**Comment by rewanthtammana** [comment]
Closing the PR. I was working on the outdated code. I compiled the binaries on outdated code, then pulled the code from main branch but forgot to build the binaries after pull.

The query in the documentation is working.
image

**Comment by pxp928** [comment]
@rewanthtammana Cool. Yea I was like this doesn't seems right!

**Comment by rewanthtammana** [comment]
@pxp928 As I went further, I started finding more commands that weren't working. I felt something's not right & realized I didn't build after fetch. I will been more cautious going forward 😁

**Pull request closed without merging**